core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -1389,6 +1408,16 @@ public void setClientData(BedrockClientData data) {
         this.inputCache.setInputMode(org.cloudburstmc.protocol.bedrock.data.InputMode.values()[data.getCurrentInputMode().ordinal()]);
     }
 
+    private void setJumpLocks(boolean lock) {
+        // Locking jump input when player start gliding so that player won't be able to stop gliding midair.
+        UpdateClientInputLocksPacket packet = new UpdateClientInputLocksPacket();
+        packet.setLockComponentData(lock ? 64 : 0); // 64 is corresponding to jump input.
```

Comment:
64 is likely supposed to be binary input. Let's convert these numbers to their 0x00b formats?